liveschool.ex

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env elixir
+
+LiveSchool.Cli.main(System.argv())
```

Comment:
I'd think we'd want to convert all of the files (or a list of them), so a mix task to run this cli module would probably be the best bet.